src/test/java/StepDefinition/DashboardTestSteps.java

```diff
@@ -0,0 +1,69 @@
+package StepDefinition;
+
+import com.ui.DashboardPage;
+import com.ui.LoginReportPortalPage;
+import com.ui.WidgetPage;
+import cucumber.TestContext;
+
+import io.cucumber.java.en.Given;
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
+import org.testng.Assert;
+import utilities.ReadPropertyFile;
+
+import java.io.IOException;
+
+public class DashboardTestSteps {
+
+    TestContext testContext;
+    LoginReportPortalPage loginReportPortalPage;
+    DashboardPage dashboardPage;
+    WidgetPage widgetPage;
+    public ReadPropertyFile fr = new ReadPropertyFile();
+
+    public DashboardTestSteps(TestContext context) throws IOException {
+        testContext = context;
+        loginReportPortalPage = testContext.getPageObjectManager().getLoginReportPortalPage();
+        dashboardPage = testContext.getPageObjectManager().getDashboardPage();
+        widgetPage = testContext.getPageObjectManager().getWidgetPage();
+    }
+
+    @Given("User login the report portal")
+    public void userLogin() throws Exception {
+        loginReportPortalPage.loginToReportPortal(fr.getPropertyValue("reportPortal"), fr.getPropertyValue("username1"), fr.getPropertyValue("password1"));
+    }
```

Comment:
Redundant, you already have this step in LoginTestStep, use it.
@When("User login the report portal with valid username {string} and password {string}")